CIP-0043/README.md

```diff
@@ -78,51 +79,62 @@ functions, these are:
 2. An input to verify (either the message itself, or a hash);
 3. A signature.
 
-The two different schemes have different expectations of each argument and how
-it is encoded into a `BuiltinByteString`. We describe these below.
+The two different schemes handle deserialization internally: specifically, there
+is a distinction made between 'external' representations, which are expected as
+arguments, and 'internal' representations, used only by the implementations
+themselves. This creates different expecations for each argument for both of
+these schemes; we describe these below.
 
-For the ECDSA signature scheme, we have the following requirements:
+For the [ECDSA signature
+scheme](https://en.bitcoin.it/wiki/Elliptic_Curve_Digital_Signature_Algorithm),
+the requirements are as follows. Note that these differ from the serialization
+used by Bitcoin.
 
+* The verification key must correspond to the _(x, y)_ coordinates of a point 
```

Comment:
This bullet point and the next one might confuse the reader, as we say that a verification key "must correspond" to two different format. I would differentiate this bullet point with the next one using phrasing as follows: 
```
* A verification key corresponds to the _(x, y)_ coordinates of a point in the SECP256k1 curve, 
where _x, y_ are unsigned integers in big-endian form.
* The serialised (compressed) verification key format accepted by the builtins must correspond [...]
```